src/utils/zod-error-parser.ts

```
@@ -0,0 +1,327 @@
+      };
+    }
+
+    // Handle other error types
+    const message =
+      error instanceof Error ? error.message : 'An unexpected error occurred';
+    return {
+      message,
+      isValidationError: false,
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)